components/infobox/wikis/rocketleague/infobox_league_custom.lua

```diff
@@ -133,9 +139,34 @@ function CustomLeague:createLiquipediaTierDisplay(args)
 
 	content = content .. '[[Category:' .. tierDisplay .. ' Tournaments]]'
 
+	-- Psyonix icon
+	if CustomLeague:containsPsyonix('organizer') then
+		content = content .. ' ' .. string.format(_PSYONIX_ICON, 'organized')
+	elseif CustomLeague:containsPsyonix('sponsor') then
+		content = content .. ' ' .. string.format(_PSYONIX_ICON, 'sponsored')
+	end
+
 	return content
 end
 
+function CustomLeague:liquipediaTierHighlighted()
+	if (
+		String.isNotEmpty(_league.args.liquipediatiertype) or
+		tonumber(_league.args.liquipediatier) ~= _TIER_1
+	) then
```

Comment:
if you want you can make it 1 line
should be 112 chars
the lua check allows 121 to my knowledge

Reply:
I think it looks ok as two lines